receiver/varnishreceiver/config.go

```diff
@@ -27,15 +28,21 @@ import (
 type Config struct {
 	scraperhelper.ScraperControllerSettings `mapstructure:",squash"`
 	Metrics                                 metadata.MetricsSettings `mapstructure:"metrics"`
-	WorkingDir                              string                   `mapstructure:"working_dir"`
+	InstanceName                            string                   `mapstructure:"instance_name"`
 	ExecDir                                 string                   `mapstructure:"exec_dir"`
 }
 
 // Validate validates the config.
 func (c *Config) Validate() error {
-	if c.WorkingDir != "" {
-		if _, err := os.Stat(c.WorkingDir); err != nil {
-			return fmt.Errorf(`"working_dir" does not exists: %w`, err)
+	if c.InstanceName != "" {
+		if _, err := os.Stat(c.InstanceName); err != nil {
+			return fmt.Errorf(`"instance_name" does not exists: %w`, err)
+		} else {
```

Comment:
I don't think this needs to be in an else statement as the if statement returns.

Reply:
fixed but now outdated